_includes/service_card.html

```diff
@@ -1,11 +1,11 @@
 {% if include.reversed %}
-  {% assign margin = 'mr-12' %}
+  {% assign margin = 'md:mr-12' %}
```

Comment:
For those not familiar with Tailwind, the prefix makes the class apply from that breakpoint onwards. So here, `mr-12` (margin-right) will be applied from medium to extra large sizes.